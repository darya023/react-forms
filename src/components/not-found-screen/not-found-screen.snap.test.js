import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router';
import {createMemoryHistory} from 'history';
import NotFoundScreen from './not-found-screen';

it(`NotFoundScreen should render correctly`, () => {
  const history = createMemoryHistory();
  const {container} = render(
      <Router history={history}>
        <NotFoundScreen />
      </Router>
  );
  expect(container).toMatchSnapshot();
});
