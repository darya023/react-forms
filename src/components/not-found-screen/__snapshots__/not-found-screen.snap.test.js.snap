// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <a
        class="link"
        href="/"
      >
        На главную
      </a>
    </header>
    <h2>
      Страница не найдена
    </h2>
  </div>
</div>
`;
